<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-FN1AS96S" failures="0" tests="10" name="tests.EndtoEndExtent" time="0.468" errors="1" timestamp="2024-06-19T00:47:27 IST" skipped="9">
  <testcase classname="tests.EndtoEndExtent" name="loginFuctionality" time="0.001">
    <skipped/>
  </testcase> <!-- loginFuctionality -->
  <testcase classname="tests.EndtoEndExtent" name="beforeTest" time="0.467">
    <error message="Cannot invoke &quot;org.openqa.selenium.WebDriver.manage()&quot; because &quot;this.driver&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebDriver.manage()" because "this.driver" is null
at tests.EndtoEndExtent.beforeTest(EndtoEndExtent.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:358)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- beforeTest -->
  <testcase classname="tests.EndtoEndExtent" name="categoriesDisplayed" time="0.000">
    <skipped/>
  </testcase> <!-- categoriesDisplayed -->
  <testcase classname="tests.EndtoEndExtent" name="navigateToNextPage" time="0.000">
    <skipped/>
  </testcase> <!-- navigateToNextPage -->
  <testcase classname="tests.EndtoEndExtent" name="contactDemo" time="0.000">
    <skipped/>
  </testcase> <!-- contactDemo -->
  <testcase classname="tests.EndtoEndExtent" name="addCartDemo" time="0.000">
    <skipped/>
  </testcase> <!-- addCartDemo -->
  <testcase classname="tests.EndtoEndExtent" name="deleteItemFromCart" time="0.000">
    <skipped/>
  </testcase> <!-- deleteItemFromCart -->
  <testcase classname="tests.EndtoEndExtent" name="checkoutProcess" time="0.000">
    <skipped/>
  </testcase> <!-- checkoutProcess -->
  <testcase classname="tests.EndtoEndExtent" name="addProductToCartWithoutLogin" time="0.000">
    <skipped/>
  </testcase> <!-- addProductToCartWithoutLogin -->
  <testcase classname="tests.EndtoEndExtent" name="checkoutWithoutLogin" time="0.000">
    <skipped/>
  </testcase> <!-- checkoutWithoutLogin -->
</testsuite> <!-- tests.EndtoEndExtent -->
